packages/core/crates/core-strategy/README.md

```diff
@@ -34,14 +34,15 @@ The constant `k` can be also set to a value > 1, which will make the strategy to
 but it would keep the same asymptotic properties.
 Per default `k` = 1.
 
-The strategy starts acting only after at least 10 network size samples were gathered, which means
-it does not start opening/closing channels earlier than 10 minutes after the node start.
+The strategy starts acting only after at least `min_network_size_samples` network size samples were gathered, which means
+it does not start opening/closing channels earlier than `min_network_size_samples` number of minutes after the node has started.
 
 ### Default parameters
 
 - `network_quality_threshold`: 0.5
 - `new_channel_stake`: 10 HOPR
 - `minimum_node_balance`: 10 HOPR
+- `min_network_size_samples`: 10
```

Comment:
The SAM window / node in the heartbeat cycle config would be good here.

Reply:
Why in strategy config? It should be rather in network/pinger config imho.

Reply:
Let's forget abiut the network object though, that's a relic. Immediate current state will be polled from the libp2p component or some form of thinner wrapper. The windowing of some value is purely strategy interest... it shouldn't live outside of it.

Reply:
I would argue that the strategy should be as state-less as possible, this way the strategy would need to carry the entire peer table with the quality windows. Especially looking into the future when CT might become (partially) part of `hoprd`, the SMA quality values should be available globally IMHO.